.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*.json]
+indent_style = space
+indent_size = 2
+
+[*.brs]
```

Comment:
Can keep the JSON but BS and BRS don't and should not appear in this project.

Reply:
Done